Makefile

```
@@ -1,48 +1,116 @@
 #! /usr/bin/make -f
+
 SHELL:=/bin/bash
 
+define pod
+
+=pod
+
+=head1 SYNOPSIS
+
+  make [-n] key=I<trello_key> token=I<trello_token> board=I<board_id> moment=I<moment_name> <command>
+  where command is one of: import ${board}.json thumbnails  ${board}.ttl moment
+
+This Makefile is used create a dams moment from a trello board.  The commands allow for the creation of a number of items needed for the momment
+
+=head1 COMMANDS
+
+=over 4
+
+=item B<make import>
+
+Imports all the required data from the trello board.  This
+
+=back
+
+=head1 OPTIONS
+
+=over 4
+
+=item B<key=I<trello_key>>
+
+This is the key for API access to the trello key. You can get this from....
+
+=item B<token=I<trello_token>>
+
+This is the token for API access to the trello key. You can get this from....
+
+=back
+
+=head1 FILE STRUCTURE
+
+The following diagram shows an example file structure.
+
+- moments
+		- cats
+    - cats_moment.ttl (the moment description)
+			- cats.ttl (new part)
+	- pY20Yz5x.json (This is the downloaded board)
+	- pY20Yz5z.ttl ( This is created from trello to moment )
+	- cats
+		- cats.json ( converted from ../pY20Yz5z.ttl with RIOT )
+			- Z4444 (a card)
+					- image_name.jpg
+
+=cut
+
+endef
+
 # use make riot=path/to/riot if not in your $PATH
 riot:=riot
 
 key:=supply_on_cmd
 token:=supply_token_on_cmd
 
-#board:=NgVOlKPZ
-#moment:=chardonney
-
-board:=K2CNYMZP
-moment:=jop
+board:=pY20Yz5x
+moment:=cats
 
 trello=http https://api.trello.com/1/boards/${board}/ key==${key} token==${token}
 
-import:${board}/board.json
 
-files:=$(patsubst %,${board}/%.json,lists cards)
+.PHONY: INFO check
+
+INFO::
+	@pod2usage -exit 0 ${MAKEFILE_LIST}
 
-.PHONY:files
-files:${files}
+check::
+	@podchecker ${MAKEFILE_LIST}
 
-${board}/board.json:
-	[[ -d ${board} ]] || mkdir ${board};\
-	${trello} lists==all cards==all card_attachments==true | jq . > $@
+import:${board}.json thumbnails
 
-thumbnails: ${board}/board.json
-	[[ -d ${moment} ]] || mkdir ${moment};\
+${board}.json:
+	[[ -d ${moment} ]] || mkdir moments/${moment}; \
+	${trello} lists==all cards==all card_attachments==true | jq . > moments/${moment}/$@
```

No `moments` directory  (anywhere) 

```suggestion
	${trello} lists==all cards==all card_attachments==true | jq . > ${moment}/$@
```

---

Removed all instances of moments dir.